castervoice/lib/ctrl/dependencies.py

```diff
@@ -4,10 +4,15 @@
 @author: synkarius
 '''
 
-import os, sys, socket, time, pkg_resources, subprocess
+try:
+    import mock
```

Comment:
Using unit test code in production, while creative, is probably not the right solution here. Better would be to defer the save/load stuff in settings.py, or better still, just move them _out_ of settings.py while keeping the `settings.SETTINGS` structure in there.

Reply:
Thank you for that, I appreciate the feedback! I've made changes dropping utilization of unit test code.